src/App.tsx

```diff
@@ -49,10 +44,42 @@ const App: React.FC = () => {
     const owner = pathArray[3];
     const repo = pathArray[4];
     setRepoName(repo);
+    // await getOldTree(owner, repo);
+    // if (oldTree) {
     await makeRequest(owner, repo);
+    // }
   };
 
   const makeRequest = async (owner: String, repo: String) => {
+    let oldTree: oldTree[] | null = null;
+
+    // Previous Comments
+    try {
+      const res = await fetch(
+        `https://api.github.com/repos/${owner}/${repo}/contents`
+      );
+      const resJSON = await res.json();
+
+      for (const key in resJSON) {
+        const file = resJSON[key];
+        if (file["path"] === "README.md") {
+          const SHA = file["sha"];
+          const blobs = await fetch(
+            `https://api.github.com/repos/${owner}/${repo}/git/blobs/${SHA}`
+          );
+          const blobsJSON = await blobs.json();
+          const decodedBlobs = atob(blobsJSON["content"]);
+          const haveComments = decodedBlobs.match(
+            /((\[.+)\]\(\.\/.+\)\s+# .+)/g
```

Comment:
abstract this regex to make more sense?
const commentsExistRegex = /rwerewr/g etc